packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.test.tsx

```diff
@@ -174,8 +180,9 @@ describe('OverflowSet', () => {
       jest.useRealTimers();
 
       // Clean up the keytip items
-      keytipManager.keytips = [];
-      keytipManager.persistedKeytips = [];
+      keytipManager.keytips = {};
+      keytipManager.persistedKeytips = {};
+      keytipManager.inKeytipMode = false;
```

Comment:
I haven't looked at keytip code before this & don't have access to the app test pages so might be missing some context here - but I assume pressing ALT enters into keytip mode and pressing again will exit it is this correct?

Reply:
Correct, but the specific reason why this was set to be false was because in certain parts of the unit test I set the value to be true.